README.md

```diff
@@ -70,6 +70,16 @@ create isolated Python environment is recommended for this project.
    ```python
    pip install -r requirements.txt
    ```
+6. Running all the test unit tests
```

Comment:
Instead of adding this to the installation section, what do you think about adding a separate header "Testing" here? If so, you can also add "Testing" in line 39 which will help users identify where the commands for running tests are located.

```suggestion
#### Testing

Run all unit tests
```

Reply:
Done